docs/data/toolpad/concepts/building-ui.md

```diff
@@ -4,77 +4,44 @@
 
 ## Component Library
 
-In order to access a list of built in components, hover over the **Component library** vertical bar to make it expand:
+In order to access a list of built in components, hover over the **Component library** vertical bar to make it expand.
 
-{{"component": "modules/components/DocsImage.tsx", "src": "/static/toolpad/docs/concepts/building-ui/library.png", "alt": "Component library", "caption": "The Toolpad Component Library"  }}
+<video controls width="auto" height="100%" style="contain" alt="component-library">
+  <source src="/static/toolpad/docs/concepts/building-ui/component-library.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
 
-Currently supported components:
+---
 
-- [Button](https://mui.com/material-ui/react-button/) - Use a button to capture user actions through a click.
-- Image - Display different types of images.
-- [DataGrid](https://mui.com/x/react-data-grid/) - Use the MUI X data grid component and easily render various types of tabular data.
-- [Text Field](https://mui.com/material-ui/react-text-field/) - Capture text from user input.
-- [Date Picker](https://mui.com/x/react-date-pickers/date-picker/) - Select a date from a date picker.
-- File Picker - Select a file from the file system.
-- [Text](https://mui.com/material-ui/react-typography/) - Present text content. Can be either plain text, markdow or links.
-- [Select](https://mui.com/material-ui/react-select/) - Capture user input from a list of options.
-- [Paper](https://mui.com/material-ui/react-paper/) - Provide a visual differantation for your components
-- [Tabs](https://mui.com/material-ui/react-tabs/) - A tabs strip that can be used to shift between different views.
-- [Container](https://mui.com/material-ui/react-container/) - A wrapper element that can be used to hide or show its children.
-- List - An array of components presented as a repeating list.
+The list of built-in components available in Toolpad can be found [here](/toolpad/reference/components/) and we are always adding more. To make it easier for us to understand what should be added next, please help by [upvoting](https://github.com/mui/mui-toolpad/labels/waiting%20for%20%F0%9F%91%8D) components.
```

Comment:
I trued to use more direct language instead of "can be found", "should be added", "by upvoting"... It feels more [conversational](https://www.notion.so/mui-org/Writing-style-guide-2a957a4168a54d47b14bae026d06a24b?pvs=4#8852c4ad7c994c06b07358bbc2885188).

```suggestion
Toolpad has many [built-in components](/toolpad/reference/components/) and we are always adding more. To make it easier for us to understand what to add next, please [upvote](https://github.com/mui/mui-toolpad/labels/waiting%20for%20%F0%9F%91%8D) the components you need, or open a [feature request](https://github.com/mui/mui-toolpad/issues/new?assignees=&labels=status%3A+needs+triage&projects=&template=2.feature.yml).
```